common/internal-types/src/channels.rs

```diff
@@ -251,19 +272,21 @@ impl ChannelChange {
     }
 }
 
-/// A wrapper around [`ChannelEntry`] representing a Channel that is corrupted.
+/// A wrapper around [`Hash`] representing a Channel that is corrupted.
 #[derive(Copy, Clone, Debug, PartialEq)]
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
-pub struct CorruptedChannelEntry(ChannelEntry);
+pub struct CorruptedChannelEntry(Hash);
```

Comment:
Maybe we can use the `ChannelId` alias instead of the `Hash` type directly. It makes it more flexible for the future.